### PR TITLE
[Cache] add Redis options compression and tcp_keepalive

### PR DESCRIPTION
**Pull request by nicolas-grekas**
Since https://github.com/symfony/symfony/pull/27646

### CODE REVIEW

**Review by HeahDude — APPROVED** [review]

File: components/cache/adapters/redis_adapter.rst [hunk 1]
```diff
@@ -128,6 +134,10 @@ Available Options
     Specifies the delay (in milliseconds) between reconnection attempts in case the client
     loses connection with the server.
 
+``tcp_keealive`` (type: ``int``, default: ``0``)
```
Comment (greg0ire) [inline components/cache/adapters/redis_adapter.rst]: You missed a `p` here

File: components/cache/adapters/redis_adapter.rst [hunk 2]
```diff
@@ -128,6 +134,10 @@ Available Options
...
+    Specifies the TCP-keepalive timeout of the connection. This requires phpredis v4 or
```
Comment (javiereguiluz) [inline components/cache/adapters/redis_adapter.rst]: I guess the timeout is in seconds, but we should specify it. Thanks!

```rst
Specifies `TCP keepalive`_ timeout of the connection in seconds. This requires phpredis v4 or higher and a TCP-keepalive enabled server.

.. _`TCP keepalive`: https://en.wikipedia.org/wiki/Keepalive
```

File: components/cache/adapters/redis_adapter.rst (hunk 1 above)
Comment (greg0ire) [reply components/cache/adapters/redis_adapter.rst]: And in the commit message :P

**Comment by nicolas-grekas** [comment]
updated thanks :)

**Review by greg0ire — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Nicolas.

**Pull request merged**